docs/examples/UserGuide/osmmakie.jl

```diff
@@ -0,0 +1,34 @@
+using Tyler, GLMakie, OSMMakie, LightOSM, TileProviders
+
+area = (
+    minlat = 51.50, minlon = -0.0921, # bottom left corner
+    maxlat = 51.52, maxlon = -0.0662 # top right corner
+)
+
+download_osm_network(:bbox; # rectangular area
+    area..., # splat previously defined area boundaries
+    network_type=:drive, # download motorways
+    save_to_file_location="london_drive.json"
+);
+
+osm = graph_from_file("london_drive.json";
+    graph_type=:light, # SimpleDiGraph
+    weight_type=:distance
+)
+
+download_osm_buildings(:bbox;
+    area...,
+    metadata=true,
+    download_format=:osm,
+    save_to_file_location="london_buildings.osm"
+);
+
+# load as Buildings Dict
+buildings = buildings_from_file("london_buildings.osm");
+# Google + OSM
+provider = TileProviders.Google(:satelite)
+m = Tyler.Map(london; provider=provider, coordinate_system=Tyler.wgs84)
```

Comment:
the problem of CI failing is that `London` is not defined in this file.  Every file runs in his own scope.

Reply:
Thanks :) The latest error is due to: https://github.com/JuliaGeo/MapTiles.jl/pull/21